test/IECoreImage/ImageWriterTest.py

```diff
@@ -818,6 +818,27 @@ def testEXRStringArrayMetadata( self ):
 
 		self.assertEqual( imgNew.blindData()["foobar"], IECore.StringVectorData( ["abc", "def", "ghi"] ) )
 
+	def testLargeNonlinearImage( self ) :
+
+		# 4k Super 35
+		dataWindow = imath.Box2i(
+			imath.V2i( 0, 0 ),
+			imath.V2i( 4095, 3111 )
+		)
+
+		imgOrig = self.__makeFloatImage( dataWindow, dataWindow )
+
+		w = IECore.Writer.create( imgOrig, "test/IECoreImage/data/png/output.png" )
+		self.assertEqual( type(w), IECoreImage.ImageWriter )
```

Comment:
assertIsInstance perhaps?

Reply:
I just copied that from the JPG test right above this one. Not sure its even necessary tbh.

Reply:
Hmm, nope I copied it from the other newly added test, which was copied from said JPG test... odd. I'll just remove this line.

Reply:
Wrong again, I copied it from the TIFF writer test a few tests above this one... Anyhow, addressed in 19ed560 along with the others in these 2 test files.